js/obs/obsHandler.js

```diff
@@ -165,9 +165,20 @@ class OBSHandler extends Handler {
         if (filterIndex === -1) {
           filterIndex = triggerData.indexOf('Filter');
         }
-        if (filterIndex === -1) {
+        var urlIndex = triggerData.indexOf('url');
+        if (urlIndex === -1) {
+          urlIndex = triggerData.indexOf('URL');
+        }
+        if (urlIndex === -1) {
+          urlIndex = triggerData.indexOf('Url');
+        }
```

Comment:
I split these out to support different cases of url due to the similar case above for `Filter` versus `Filter`. Would it be better to simply do
```javascript
var urlIndex = triggerData.toLowerCase().indexOf('url');
```
or something along these lines?

Reply:
I'll merge this in and see if I can add a solution for both filter and url (if not just a method to make getting the index value easier).